projects/packages/waf/phpunit.xml.dist

```diff
@@ -1,14 +1,21 @@
-<phpunit bootstrap="tests/php/bootstrap.php" backupGlobals="false" colors="true" convertDeprecationsToExceptions="true">
-	<testsuites>
-		<testsuite name="main">
-			<directory prefix="test" suffix=".php">tests/php</directory>
-		</testsuite>
-	</testsuites>
-	<filter>
-		<whitelist processUncoveredFilesFromWhitelist="false">
-			<!-- Better to only include "src" than to add "." and then exclude "tests", "vendor", and so on, as PHPUnit still scans the excluded directories. -->
-			<!-- Add additional lines for any files or directories outside of src/ that need coverage. -->
-			<directory suffix=".php">src</directory>
-		</whitelist>
-	</filter>
+<?xml version="1.0"?>
```

Comment:
Do you happen to know why the additions to this file were needed?

Reply:
Sorry, I should have commented on that from the beginning. Yes, PHPUnit prompted me to update the config since it was apparently using an outdated format. It also offered to migrate it for me, to which I agreed. 

I then only made sure that the comment is in again and formatted the very long line beginning with `<phpunit` so that it better fits onto the screen (was in one line before).